homeassistant/components/nordpool/sensor.py

```diff
@@ -56,7 +56,7 @@ def get_prices(
     current_time = dt_util.utcnow()
     previous_time = current_time - timedelta(hours=1)
     next_time = current_time + timedelta(hours=1)
-    LOGGER.debug("Price data: %s", data)
+    # LOGGER.debug("Price data: %s", data)
```

Comment:
Why is this commented out?

Reply:
Thanks, I remove it while debugging the code change as it's quite verbose. Forgot to reset it back so need to fix that 👍